waspc/cli/Command/BashCompletion.hs

```diff
@@ -0,0 +1,48 @@
+module Command.BashCompletion
+  ( bashCompletion,
+  )
+where
+
+import Command (Command)
+import Control.Monad.IO.Class (liftIO)
+import qualified Paths_waspc
+
+bashCompletion :: [String] -> Command ()
+bashCompletion args = case args of
+  [] -> listCommands
+  ["bash-completion-generate"] -> generateBashCompletionScript
+  ["bash-completion-instruction"] -> printBashCompletionInstruction
```

Comment:
These two are not completions, so I would not put them here. Let's instead call these commands (`generateBashCompletionScript` and `printBashCompletionInstruction`) directly from the `Main.hs`, and you would also add them to `Command/Call.hs`